src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotJVMCIRuntime.java

```diff
@@ -240,7 +240,7 @@ public enum Option {
                 "name (a stub) or fully qualified name (an nmethod) contains this option's value as a substring."),
         ForceTranslateFailure(String.class, null, "Forces HotSpotJVMCIRuntime.translate to throw an exception in the context " +
                 "of the peer runtime. The value is a filter that can restrict the forced failure to matching translated " +
-                "objects. See HotSpotJVMCIRuntime.postTranslation for more details. This option exists soley to test " +
+                "objects. See HotSpotJVMCIRuntime.postTranslation for more details. This option exists solely to test " +
                 "correct handling of translation failure."),
```

Comment:
Should this be plural?

```suggestion
                "correct handling of translation failures."),
```

Reply:
Thanks. Suggested change applied.